assets/src/components/parts/janus-dropdown/Dropdown.tsx

```diff
@@ -322,7 +322,7 @@ const Dropdown: React.FC<PartComponentProps<DropdownModel>> = (props) => {
               const sSelectedIndex = changes[`stage.${id}.selectedIndex`];
               if (sSelectedIndex !== undefined) {
                 const stateSelection = Number(sSelectedIndex);
-                if (selectedIndex !== stateSelection) {
+                if (selectedIndex !== stateSelection || stateSelection === -1) {
```

Comment:
This change was needed to be done in Context_Change and  not in State_Change. I did that by mistake in PR - https://github.com/Simon-Initiative/oli-torus/pull/2153